src/EnergyPlus/ZoneContaminantPredictorCorrector.cc

```diff
@@ -1316,9 +1316,6 @@ namespace ZoneContaminantPredictorCorrector {
 
         } // ContControlledZoneNum
 
-        // Get the Hybrid Model setting inputs
-        GetHybridModelZone();
-
```

Comment:
Whoa.  This wasn't the fix that I expected.  But OK!

Reply:
Not 100% certain this is the proper fix. We could add a flag to make sure this only runs once, but I can't see any reason why this needs to be here when it's already getting called [here](https://github.com/NREL/EnergyPlus/blob/7a131682e99198678e82db06e8d60cb4097b267e/src/EnergyPlus/ZoneTempPredictorCorrector.cc#L1777-L1778) in `GetZoneAirSetPoints`. I suppose it's possible that one could do a hybrid model on CO2 without any HVAC? Then would `GetZoneAirSetPoints` never be called? Hmmm.

Reply:
`GetZoneAirSetPoints` is called from `ManageZoneAirUpdates` which will *always* get called, so I think this is OK.

Reply:
I'd suggest moving it even higher to ensure that it gets called once, and only once, every simulation.

Reply:
Any suggestions? A quick scan says the only things that need to be done before this are Zones and ZoneAirMassFlow.EnforceZoneMassBalance but I may have missed something. Well, schedules, too.

Reply:
I'm not sure but `GetZoneAirSetPoints` already has a `GetZoneAirStatsInputFlag`, would this flag make sure that `GetHybridModelZone` under this be called only once?